pages/tidal tools/analyze_database.md

```diff
@@ -133,3 +140,25 @@ You need to install Docker in order to complete the database analysis. This is b
 
 ##  What about security?
 The entire analysis takes place locally on your machine. The only data that is captured and sent from the analysis are the results of the analysis and metadata. No application data, source code, files or the contents of any files on your machine are ever copied or sent anywhere.
+
+## Advanced Configuration
+
+To use Oracle features included only in the Oracle Standard Edition (SE) license, you
+can set the `analyze_workload` flag to `false` in your configuration file. For example:
```

Comment:
It's not a **flag**, but **property** or **option**. WDYT?

Reply:
yep fair I'll switch to property.